.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 node_js:
   - "0.10"
+before_install: npm install -g grunt-cli
```

Comment:
can we just add `grunt-cli` to the package.json and then travis will install it automatically without the need for this line